src/testUtils/TestRig.react.js

```diff
@@ -57,17 +59,15 @@ export default class TestRig {
     var element = this.domNode.find(selector)
     this.fillInElement(element, value)
   }
-  toggleCheckbox(element) {
-    element.checked = !(element.checked)
-    TestUtils.Simulate.change(element)
-  }
-  toggleRadioButton(element) {
-    element.checked = !(element.checked)
-    TestUtils.Simulate.change(element)
+  toggleElement(selector) {
+    var element = this.domNode.find(selector)
```

Comment:
@andela-tkomolafe You can move the `[0]` to the end of this line so that you don't have to call it in three places below.